Doc/whatsnew/3.12.rst

```diff
@@ -329,6 +329,11 @@ Removed
 
 .. _aiosmtpd: https://pypi.org/project/aiosmtpd/
 
+* ``asynchat`` and ``asyncore`` have been removed
+  according to the schedule in :pep:`594`,
+  having been deprecated in Python 3.6.
+  Use :mod:`asyncio` instead.
```

Comment:
It's good to add a "(Contributed by X in Y)" line, to easily reference the relevant issue.

If you want to omit your name, "(Contributed by Y)" is also good.

Reply:
Done, thank you!